src/Sylius/Bundle/ResourceBundle/Form/Type/ResourceTranslationsType.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\Form\Type;
+
+use Sylius\Component\Resource\Provider\AvailableLocalesProviderInterface;
+use Sylius\Bundle\ResourceBundle\Form\EventSubscriber\ResourceTranslationsSubscriber;
+use Sylius\Component\Resource\Provider\LocaleProviderInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class ResourceTranslationsType extends AbstractType
+{
+    /**
+     * @var LocaleProviderInterface
+     */
+    protected $localeProvider;
+
+    /**
+     * @var AvailableLocalesProviderInterface
+     */
+    protected $availableTranslationLocalesProvider;
+
+
+    /**
+     * @param LocaleProviderInterface $localeProvider
+     * @param AvailableLocalesProviderInterface $availableTranslationLocalesProvider
+     */
+    public function __construct(LocaleProviderInterface $localeProvider, AvailableLocalesProviderInterface $availableTranslationLocalesProvider)
+    {
+        $this->localeProvider = $localeProvider;
+        $this->availableTranslationLocalesProvider = $availableTranslationLocalesProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $locales = $this->availableTranslationLocalesProvider->getAvailableLocales();
+        $localesWithRequirement = array();
+        foreach($locales as $locale) {
+            $localesWithRequirement[$locale] = false;
+
+            if($this->localeProvider->getDefaultLocale() === $locale) {
+                $localesWithRequirement[$locale] = true;
+            }
+        }
```

Comment:
Maybe logic for selecting required locales could be extracted to another service, so we could easily replace it?